### ISSUE TITLE
Implement keyboard navigation in the Clarity datagrid

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#3987]
## Summary
Provide keyboard navigation for accessing every control in the data grid

* What is the change?
Implement keyboard navigation with arrows and common keyboard shortcuts for the advanced actions
* Why should it go in Clarity?
Provides important accessibility features for the data grid component

* Does this change impact existing behaviors? If so how?
No
* If this change introduces a new behavior, is this behavior accessible?
Yes

## Use case

As a keyboard user of the data grid I want to be able to navigate to each cell with the arrow keys and perform advanced actions with keyboard shortcuts

## Examples

https://www.ag-grid.com/example.php#/
http://whatsock.com/tsg/Coding%20Arena/ARIA%20Data%20Grids/ARIA%20Data%20Grid%20(Dynamic)/demo.htm

## Workarounds
No

**Comment by gnomeontherun** [comment]
This is a duplicate of https://github.com/vmware/clarity/issues/2356 and I'm going to close this as such. Please double check our list of issues before posting something new, and better to comment on an issue as well.

**Comment by bashibozuk** [comment]
@gnomeontherun #2356 is closed but not implemented which makes me think that it is not something that you are working on or you will release in near future ?

**Comment by gnomeontherun** [comment]
#2356 is not closed, but it is not something we're prioritizing right now because the datagrid is still navigable by screen readers and keyboard users. It just doesn't have all of the functionality we'd like to offer. The other challenge is that as the datagrid changes, the implementation of this feature would change so it will be very unlikely we can backport this fix.
